src/connection.c

```diff
@@ -30,6 +30,12 @@ typedef enum RECEIVE_FRAME_STATE_TAG
     RECEIVE_FRAME_STATE_FRAME_DATA
 } RECEIVE_FRAME_STATE;
 
+typedef struct ON_CONNECTION_CLOSED_EVENT_SUBSCRIPTION_TAG
+{
+    ON_CONNECTION_CLOSE_RECEIVED on_connection_close_received;
+    void* context;
+} ON_CONNECTION_CLOSED_EVENT_SUBSCRIPTION;
```

Comment:
>ON_CONNECTION_CLOSED_EVENT_SUBSCRIPTION [](start = 2, length = 39)

Curious naming, I thought this was a function point, but it is a structure.

Reply:
Yeah, /poetry. Noted, though about it, but decided to leave as is :-)

---
In reply to: [192261066](https://github.com/Azure/azure-uamqp-c/pull/240#discussion_r192261066) [](ancestors = 192261066)